package.json

```diff
@@ -6,15 +6,19 @@
   "scripts": {
     "start": "jekyll serve",
     "build": "NODE_ENV=production ./node_modules/webpack/bin/webpack.js -p",
+    "lint": "standard",
     "watch": "./node_modules/webpack/bin/webpack.js --watch"
   },
+  "repository": "speedtracker/speedtracker",
   "author": "Eduardo Bouças <mail@eduardoboucas.com> (https://eduardoboucas.com)",
   "license": "MIT",
   "devDependencies": {
     "babel-loader": "^6.2.5",
     "babel-preset-es2015": "^6.13.2",
     "babel-preset-es2016": "^6.11.3",
     "babel-preset-react": "^6.11.1",
+    "eslint-plugin-standard": "^3.0.1",
```

Comment:
`eslint-plugin-standard` is not necessary if you're already installing `standard`. You can remove this!